src/libs/antares/study/parts/short-term-storage/container.cpp

```diff
@@ -78,16 +78,10 @@ bool STstorageInput::loadSeriesFromFolder(const std::string& folder)
 
     for (auto& cluster : storagesByIndex)
     {
-        const std::string buffer = folder + SEP + cluster->properties.name + SEP + "series.txt";
-        /* ret = series->series.loadFromCSVFile(buffer, 1, HOURS_PER_YEAR, &s.dataBuffer) && ret; */
-        cluster->loadSeries(folder);
+        const std::string buffer(folder + SEP + cluster->properties.name + SEP);
```

Comment:
1. Use id, not the name. Names may contain special characters such as \, /, etc. that can't be used for file names. You need to introduce the id for that, and use function `TransformNameIntoID`
2. Don't pass the last separator to the function, let cluster->loadSeries handle it.
```suggestion
        const std::string buffer(folder + SEP + cluster->id);
```